### PR TITLE
CE-3788: Download new translations

### PR DESCRIPTION
**Pull request by lukaszkonieczny**

### CODE REVIEW

**Comment by lukaszkonieczny** [comment]
cc @Wikia/spitfires

File: extensions/wikia/CommunityPage/CommunityPage.i18n.php [hunk 1]
```diff
@@ -667,7 +658,7 @@
 	'communitypage-uncategorizedpages-description' => 'Dodaj kategorie do stron. Poprawne określenie kategorie pomoże Ci w organizacji wikii!',
 	'communitypage-wantedpages-title' => 'Potrzebne strony',
 	'communitypage-wantedpages-description' => 'Istnieją redlinki kierujące do tych stron, ale te strony nie istnieją! Stwórz artykuł, albo przekieruj do właściwego artykułu, aby pomóc czytelnikom poruszać się po twojej społeczności.',
-	'communitypage-lastrevision' => 'Użytkownik $1 edytował $2',
+	'communitypage-lastrevision' => 'Użytkownika $1 edytował $2',
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/CommunityPage.i18n.php]: Again?

File: extensions/wikia/CommunityPage/CommunityPage.i18n.php [hunk 2]
```diff
@@ -744,7 +734,7 @@
 	'communitypage-wantedpages-title' => 'Páginas pedidas',
 	'communitypage-wantedpages-description' => 'Há links apontando para essas páginas, mas elas não existem! Crie um artigo ou o redirecione para o artigo correto para ajudar os leitores a navegar em sua comunidade.',
 	'communitypage-lastrevision' => '$1 editou às $2',
-	'communitypage-noofviews' => '$1 visualizações',
+	'communitypage-noofviews' => '$ 1 visualizações',
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/CommunityPage.i18n.php]: whyy? :)

**Comment by kamilkoterba** [comment]
LGTM

**Comment by kamilkoterba** [comment]
Tested LGTM

**Pull request merged**